pypsa/io.py

```diff
@@ -1088,18 +1094,19 @@ def import_from_pandapower_net(network, net, extra_line_data=False):
     else:
         s_nom = net.trafo.sn_mva.values/1000.
```

Comment:
s_nom is given in MVA, net.trafo.sn_mva is already in MVA, no need to divide by 1000.

Reply:
fixed now thanks